src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -378,7 +378,11 @@ public static Config BaseConfig
         private static string GetCallerName()
         {
             var @this = typeof(MultiNodeSpec).Name;
+#if CORECLR
+            var trace = new StackTrace(new Exception(), false);
```

Comment:
I know, but you can't call StackTrace() without parameters in `NetCore` anymore.